server/paymentProviders/transferwise/index.ts

```diff
@@ -67,7 +68,57 @@ async function payExpense(connectedAccount, payoutMethod, expense): Promise<any>
   return { quote, recipient, transfer, fund };
 }
 
+async function getRequiredBankInformation(collective, currency: string): Promise<any> {
+  const cacheKey = `transferwise_required_bank_info_${collective.id}_to_${currency}`;
+  const fromCache = await cache.get(cacheKey);
+  if (fromCache) {
+    return fromCache;
+  }
+
+  const host = collective.isHostAccount ? collective : await collective.getHostCollective();
+  const [connectedAccount] = await host.getConnectedAccounts({
+    where: { service: 'transferwise', deletedAt: null },
+  });
+
+  if (!connectedAccount) {
+    throw new Error('Host is not connected to Transferwise');
+  }
+
+  const quote = await transferwise.createQuote(connectedAccount.token, {
+    profileId: connectedAccount.data.id,
+    sourceCurrency: collective.currency,
+    targetCurrency: currency,
+    targetAmount: 100,
+  });
+  const requiredFields = await transferwise.getAccountRequirements(connectedAccount.token, quote.id);
+  cache.set(cacheKey, requiredFields, 24 * 60 * 60 /* a whole day and we could probably increase */);
+  return requiredFields;
+}
+
+async function getAvailableCurrencies(collective): Promise<any> {
+  const cacheKey = `transferwise_available_currencies_${collective.id}`;
```

Comment:
Same here, great that you used a cache, let's have it at the host level if that's possible.